content/blog/2023-04-05-response-to-forbes-article-about-ethereum-classic/index.md

```diff
@@ -0,0 +1,340 @@
+
+*Ethereum Classic has largely traded like smaller cryptocurrencies, and it’s tightly correlated but more volatile than Bitcoin and Ethereum.”*
+
+### Our response:
+
+ETC will likely climb to be one of the top four blockchains in the world. This is because ETC actually does not occupy Ethereum’s position in the market. 
+
+As a proof of stake blockchain, ETH cannot be a secure base layer system. Only proof of work networks as Bitcoin and ETC may position themselves in that segment. 
+
+In the near future these things will become more defined in the industry and the great majority of the proof of stake blockchains will diminish while ETC will likely continue to climb in the ranks.
+
+**For more information:**
+
+- What Is Ethereum Classic’s Contribution to the Blockchain Industry? https://etherplan.com/2021/04/20/what-is-ethereum-classics-contribution-to-the-blockchain-industry/15717/ 
+
+## Section: Should You Invest in Ethereum Classic?
+
+### The first paragraph of this section reads:
+
+*“Given issues concerning scalability and energy consumption, as well as the continued mainstream adoption of Ethereum, it’s difficult to imagine Ethereum Classic ever gaining a foothold like Ethereum.”*
+
+### Our response:
+
+As previously described in our responses above, scalability is not an issue in the blockchain industry and energy consumption is a fallacy because proof of work actually helps the planet by accelerating the migration to renewable energy.
+
+ETH and ETC will actually not compete in them market. Ethereum will have a certain position in the market as a layer 2 scalable system and Ethereum Classic will have a different position as a base layer, highly secure blockchain. Both will be very valuable, but arguably, Ethereum Classic more valuable than Ethereum due to its true security guarantees.
+
+### The second paragraph of this section reads:
+
+*“ETH is working to improve its drawbacks, while Ethereum Classic is staying largely the same.”*
+
+### Our response:
+
+Again, Ethereum is making its drawbacks worse by constantly adding patches and fixes to its stack that increase complexity which is a major source of risk and attack vectors. On top of this, it is driving its system to centralization which is exactly opposite of what a blockchain should do.
+
+As explained before, Ethereum Classic is not staying largely the same. It is actually on par with Ethereum technologically and will always be state of the art technology.
+
+In fact, Ethereum Classic and Ethereum will both ossify in the near future, precisely because adding components, introducing fixes, and complex design to make them work is actually a huge danger to these systems.
+
+When ETC is regarded as an “immutable” system it is because of its guarantees with regards to accounts, balances, and dapps. This is its true and distinguishing value, and is only possible with the real decentralization that proof of work provides.
```

Comment:
Line 314:
```suggestion
When ETC is regarded as an “immutable” system it is because of its guarantees with regard to accounts, balances, and dapps. This is its true and distinguishing value, only possible with the real decentralization that proof of work provides.
```